packages/contracts-bedrock/justfile

```diff
@@ -193,6 +193,12 @@ validate-spacers-no-build:
 # Checks that spacer variables are correctly inserted.
 validate-spacers: build validate-spacers-no-build
 
+# Checks that the Kontrol summary dummy files have not been modified.
+# If you have changed the summary files deliberately, update the hashes in the script.
+# Use `openssl dgst -sha256` to generate the hash for a file.
```

Comment:
Can you remind me what the dummy files are and when they should be updated?

Reply:
We recently got rid of those giant summary files and instead we just have dummy files that have the right interface to be able to get the contracts to compile. You might need to update the dummy files if you edit the Kontrol proofs.